satpy/tests/reader_tests/test_seviri_l1b_nc.py

```diff
@@ -170,22 +185,22 @@ def file_handler(self, counts):
             )
 
     @pytest.mark.parametrize(
-        ('channel', 'calibration', 'use_ext_coefs'),
+        ('channel', 'key', 'calibration', 'use_ext_coefs'),
         [
             # VIS channel, internal coefficients
-            ('VIS006', 'counts', False),
-            ('VIS006', 'radiance', False),
-            ('VIS006', 'reflectance', False),
+            ('VIS006', 'ch1', 'counts', False),
```

Comment:
Is it possible to store the `ch1/9` keys in a global dictionary and look them up inside the test function? That would keep the parametrization simpler.